src/UI/page.js

```diff
@@ -207,16 +230,18 @@ function approximateFraction(x) {
 
 function getOutputScale(ctx) {
   let devicePixelRatio = window.devicePixelRatio || 1;
-  let backingStoreRatio = ctx.webkitBackingStorePixelRatio ||
-                          ctx.mozBackingStorePixelRatio ||
-                          ctx.msBackingStorePixelRatio ||
-                          ctx.oBackingStorePixelRatio ||
-                          ctx.backingStorePixelRatio || 1;
+  let backingStoreRatio =
+    ctx.webkitBackingStorePixelRatio ||
+    ctx.mozBackingStorePixelRatio ||
```

Comment:
These are all deprecated now, other than the webkit one. The rest would always just return 1 / are 1 so can be removed. Only apple is difficult